packages/vite-plugin-react-router/src/plugin.ts

```diff
@@ -0,0 +1,106 @@
+import type { Plugin, ResolvedConfig } from 'vite'
+import { mkdir, writeFile } from 'node:fs/promises'
+import { join, relative, sep } from 'node:path'
+import { sep as posixSep } from 'node:path/posix'
+import { version, name } from '../package.json'
+
+const NETLIFY_FUNCTIONS_DIR = '.netlify/functions-internal'
```

Comment:
oh, we should probably take 15 mins to update this to use the Frameworks API instead of spreading this to a brand "new" adapter

Reply:
... I think it's literally just changing this one path

Reply:
oh yeah, I always forget that _technically_ changing to use the Frameworks API is a breaking change because it won't be supported for manual deploys with [netlify-cli < 17.31.0](https://github.com/netlify/cli/pull/6737)

which is to say we should definitely use it in this brand new package

Reply:
Welp, I just changed the path and it just worked ✅